tests/test_isort.py

```diff
@@ -605,11 +605,33 @@ def test_length_sort() -> None:
     )
 
 
+def test_length_sort_straight() -> None:
+    """Test setting isort to sort straight imports on length instead of alphabetically."""
+    test_input = (
+        "import medium_sizeeeeeeeeeeeeee\n"
+        "import shortie\n"
+        "import looooooooooooooooooooooooooooooooooooooong\n"
+        "from medium_sizeeeeeeeeeeeeee import b\n"
+        "from shortie import c\n"
+        "from looooooooooooooooooooooooooooooooooooooong import a\n"
+    )
+    test_output = isort.code(test_input, length_sort_straight=True)
+    assert test_output == (
+        "import shortie\n"
+        "import medium_sizeeeeeeeeeeeeee\n"
+        "import looooooooooooooooooooooooooooooooooooooong\n"
+        "from looooooooooooooooooooooooooooooooooooooong import a\n"
+        "from medium_sizeeeeeeeeeeeeee import b\n"
+        "from shortie import c\n"
+    )
+
+
 def test_length_sort_section() -> None:
     """Test setting isort to sort on length instead of alphabetically for a specific section."""
     test_input = (
         "import medium_sizeeeeeeeeeeeeee\n"
         "import shortie\n"
+        "import datetime\n"
```

Comment:
Fixed this test as `os` and `sys` sort the same in both _alphabetical_ and _length sort_ mode. Added `datetime` that would be first in _alphabetical_, but last in _length sort_ mode.